piqe_ocp_lib/api/resources/ocp_operators.py

```diff
@@ -200,6 +200,19 @@ def get_package_ownnamespace_channel(self, package_name):
             logger.error("A OwnNamespace channel was not found for package: {}".format(package_name))
         return None
 
+    def get_yaml_from_annotation(self, channel_name):
+        """
+        A method that returns a channel object
+        :param channel_name: name of the channel 
+        :return: channel object
+        """
+        pkg_details = self.get_package_manifest("amq-streams")
+        channels_list = self.get_package_channels_list(pkg_details.metadata.name)
+
+        for channel in channels_list:
```

Comment:
Maybe do tweak the loop as follows:
target_channel = None
for channel in channels_list:
    if channel.name == channel_name:
        target_channel = channel
        break
alm = target_channel.currentCSVDesc.annotations['alm-examples']
alm_list = json.loads(alm)
return alm_list